### PR TITLE
Native touch animations for UWP and Android

### PR DESCRIPTION
**Pull request by piersdeseilligny**
This pull request adds:

* Native touch animations for UWP and Android (Tilt and Ripple)

* Updated Documentation

Resolve issue #31

### CODE REVIEW

File: TouchEffect.Droid/PlatformTouchEff.cs [hunk 1]
```diff
@@ -92,5 +136,85 @@ private void OnTouch(object sender, AView.TouchEventArgs e)
...
+        }
+
+        public bool EndRipple()
+        {
+            if (_viewOverlay != null && _viewOverlay.Pressed)
+            {
+                _viewOverlay.Pressed = false;
+                return true;
+            }
+            else
```
Comment (AndreiMisiukevich) [inline TouchEffect.Droid/PlatformTouchEff.cs]: no need "else" block, please :)

File: TouchEffect.Droid/PlatformTouchEff.cs [hunk 2]
```diff
@@ -92,5 +136,85 @@ private void OnTouch(object sender, AView.TouchEventArgs e)
...
+        public bool StartRipple(float x, float y)
+        {
+            if (_effect.NativeAnimation && _effect.AndroidRipple && _viewOverlay.Background is RippleDrawable)
+            {
+                _viewOverlay.BringToFront();
+                _ripple.SetHotspot(x, y);
+                _viewOverlay.Pressed = true;
+                return true;
+            }
+            else
```
Comment (AndreiMisiukevich) [inline TouchEffect.Droid/PlatformTouchEff.cs]: no need "else" block, please :)

**Comment by AndreiMisiukevich** [comment]
>> This branch has conflicts that must be resolved

Could you please make sure you branched from the last dev code?

**Review by AndreiMisiukevich — CHANGES_REQUESTED** [review]
please rebase onto the head of "dev" branch

**Comment by piersdeseilligny** [comment]
So rebasing has made a mess, I'm going to refork this repo and submit a new pull request. Apologies for my incompetence with git.

**Comment by AndreiMisiukevich** [comment]
> So rebasing has made a mess, I'm going to refork this repo and submit a new pull request. Apologies for my incompetence with git.

sure) thanks mate)

**Pull request closed without merging**